just-premium-htb.js

```diff
@@ -410,9 +467,9 @@ function JustPremiumHtb(configs) {
             },
             bidUnitInCents: 1, // The bid price unit (in cents) the endpoint returns, please refer to the readme for details
```

Comment:
From the https://github.com/indexexchange/just-premium-adapter/blob/development-v2/spec/support/mockResponseData.json, it looks like the price passed back is in dollar. Please change the bidUnitsInCents to 100 instead.